clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -893,3 +903,31 @@
       Out: ""
       MatchMode: Partial
       RuleId: "remove_add_arch"
+
+- Rule: rule_string
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: string
+  In: string(${option} ${args})
+  Out: string(${option} ${args})
```

Comment:
The migration rule is not needed in current pattern, as it does not change the original CMake script.
My idea is that the string variable defined by command "string" may have some CUDA specific keyword used in some Apps, if it does, we should migrated to the corresponding one in SYCL side.
You can investigate it in github code repos. If this scenario does not exist in github APP's code repo, we can skip this cmake syntax.

Reply:
Removed the rule